frontend/src/components/CircularLoader/CircularLoader.tsx

```diff
@@ -0,0 +1,11 @@
+import * as React from 'react';
+import CircularProgress from '@mui/material/CircularProgress';
+import Box from '@mui/material/Box';
+
+export default function CircularIndeterminate() {
```

Comment:
Why you use different names of components here in export and import to another file ? Here name `CircularIndeterminate` and in import we have `CircularLoader`. If it possible , I would rather use similar names in export and import